packages/edge-bundler/package.json

```diff
@@ -64,7 +64,7 @@
     "better-ajv-errors": "^1.2.0",
     "common-path-prefix": "^3.0.0",
     "env-paths": "^3.0.0",
-    "esbuild": "0.21.2",
+    "esbuild": "0.19.11",
```

Comment:
Still seeing some esbuilds with version `0.25.1` in the `package-lock.json`, example:
```
"node_modules/vite/node_modules/@esbuild/android-arm64": {
"version": "0.25.1",
...
```